netbox/templates/account/token.html

```diff
@@ -15,11 +15,6 @@
 {% block content %}
   <div class="row">
     <div class="col col-md-12">
-      {% if key and not settings.ALLOW_TOKEN_RETRIEVAL %}
```

Comment:
I think a warning message is still relevant?  They have to copy it on this form before leaving?

Reply:
The key is only shown on the form prior to creation now; it's never passed to the view afterward.